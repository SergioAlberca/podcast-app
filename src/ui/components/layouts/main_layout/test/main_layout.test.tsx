import { render } from "@/test/test_utils";

import MainLayout from "../main_layout";
import { BrowserRouter } from "react-router-dom";

describe("MainLayout component tests", () => {
  it("render component", () => {
    const { container } = render(
      <BrowserRouter>
        <MainLayout />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
