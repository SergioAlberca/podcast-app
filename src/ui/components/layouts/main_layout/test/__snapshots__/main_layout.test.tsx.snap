// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MainLayout component tests > render component 1`] = `
<div>
  <div
    class="_main-layout_bbfac7"
  >
    <header
      class="_header_a4172c"
    >
      <a
        href="/"
      >
        <h2>
          Podcaster
        </h2>
      </a>
    </header>
  </div>
</div>
`;
